casper/src/test/scala/coop/rchain/casper/MultiParentCasperMergeSpec.scala

```diff
@@ -68,7 +68,11 @@ class MultiParentCasperMergeSpec
   }
 
   it should "respect mergeability rules when merging blocks" in effectTest {
-    mergeabilityCases.map(_._2).parSequence
+    baseMergeabilityCases.map(_._2).parSequence
+  }
+
+  it should "respect mergeability rules when merging blocks containing events with peek" in effectTest {
+    peekMergeabilityCases.map(_._2).parSequence
```

Comment:
Let's add it to the previous test, maybe? Peek is not that special.

Reply:
Let's leave them separated

Reply:
I agree with Lukasz: so long as peek is not merely syntactic sugar for consume-then-produce, it merits being tested separately.